public/app/features/alerting/unified/NotificationPolicies.tsx

```diff
@@ -227,11 +230,22 @@ const AmRoutes = () => {
                 <GrafanaAlertmanagerDeliveryWarning currentAlertmanager={selectedAlertmanager} />
                 <Stack direction="column" gap={1}>
                   {rootRoute && (
-                    <NotificationPoliciesFilter
-                      receivers={receivers}
-                      onChangeMatchers={setLabelMatchersFilter}
-                      onChangeReceiver={setContactPointFilter}
-                    />
+                    <Stack direction="row" gap={1}>
```

Comment:
Perhaps we want to move this logic to the `NotificationPoliciesFilter`, passing a number of matches as a property;

1. It already has some code to determine if we have any filters (though uses query params)
1. We can then wrap the `<Button>` and the results with a `<Stack alignItems="center">`
2. There's an odd delay right now between the button disappearing and the number of matches which would be eliminated (probably because query strings are used to determine rendering of the clear button)
3. keeps the logic here a bit cleaner since it doesn't need to be concerned about rendering anything related to filters